### PR TITLE
audren: Increase queued mixed buffer count

### LINKED ISSUE #4990: Sakuna: Of Rice and Ruin (Title ID:0100B1400E8FE000)  has a sound glitch
Sakuna: Of Rice and Ruin (Title ID:0100B1400E8FE000)  has a sound glitch

this game is famous in japan.
iI think it is better to cure this bug as soon as possible.

video:https://www.youtube.com/watch?v=0HQmy_kFjHQ&feature=youtu.be

### LINKED ISSUE #5032: OceanHorn 30 vs 60 fps audio issues. 1144 build Texture Cache Rewrite feature which makes the game render.
Hello, i know this is experimental. I just wanted to report something about OceanHorn 2. On native 30FPS the audio is breaking a lot, but if you put the 60 FPS mod, and run it at those fps, the audio seems to be working properly. Also i will attach here the 60fps mod video with the audio working properly. 

I do have enable "Enable audio stretching" from yuzu options. 
Accuracy lvl high - normal and extreme. Same issue with 30 fps for those 3 options.

With native 30 fps: 
https://streamable.com/n28akj

With 60 FPS Patch
https://streamable.com/o2faof

### PR DESCRIPTION
**Pull request by ogniK5377**
Fixes issues in games that update audio a lot slower.

Closes #5032, closes #4990

### CODE REVIEW

File: src/audio_core/audio_renderer.cpp [hunk 1]
```diff
@@ -16,6 +16,7 @@
 #include "core/settings.h"
 
 namespace {
+static constexpr std::size_t MAX_QUEUED_AUDIO_BUFFERS = 16;
```
Comment (lioncash) [inline src/audio_core/audio_renderer.cpp:19]:
```suggestion
constexpr std::size_t MAX_QUEUED_AUDIO_BUFFERS = 16;
```

A newline should also be added after this line.

**Comment by bunnei** [comment]
Breaks Voez

**Comment by ActualMandM** [comment]
Also breaks Xenoblade Chronicles 2
2020-12-01_16-03-47_yuzu

**Comment by ogniK5377** [comment]
> Also breaks Xenoblade Chronicles 2
> 2020-12-01_16-03-47_yuzu

Only seems to break in multicore mode. Seems to be a separate issue which was unearthed by this PR

**Comment by Shoegzer** [comment]
I believe this may also fix a similar audio issue with Dead Cells, as the symptoms seem similar to others referenced in this PR.

**Comment by german77** [comment]
This can probably be closed since it has been superseded by #6498

**Comment by Morph1984** [comment]
Superseded by #6498

**Pull request closed without merging**